dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java

```diff
@@ -159,9 +159,9 @@ private void onRTEvent(RequestEvent event) {
         MethodMetric metric = new MethodMetric(applicationModel, event.getAttachmentValue(MetricsConstants.INVOCATION));
         long responseTime = event.getTimePair().calc();
         if (enableRt) {
-            TimeWindowQuantile quantile = ConcurrentHashMapUtils.computeIfAbsent(rt, metric,
-                k -> new TimeWindowQuantile(DEFAULT_COMPRESSION, bucketNum, timeWindowSeconds));
-            quantile.add(responseTime);
+            ConcurrentDoubleHistogram concurrentDoubleHistogram = ConcurrentHashMapUtils.computeIfAbsent(rt, metric,
```

Comment:
Is `timeWindowSeconds` here no longer needed

Reply:
let me look look

Reply:
Sliding windows are used for aggregation statistics of minimum and maximum values. Unlike the percentile algorithm above, sliding windows use thread safe algorithms